src/format-files.js

```diff
@@ -47,6 +49,11 @@ function formatFilesFromArgv({
     prettierLast,
     prettierOptions,
   }
+
+  if (eslintConfigPath) {
+    prettierESLintOptions.eslintConfig = ConfigFile.load(eslintConfigPath)
```

Comment:
Could we get a test for this? I'd be fine if you wanted to add `// istanbul ignore if` on line 52 because testing this in the unit tests would be tough, but then we could add this to the [`dist-test`](https://github.com/prettier/prettier-eslint-cli/tree/master/cli-test)s. Could you do that please? That'd be swell!